[2024-08-31T02:16:40.733+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-31T02:16:40.749+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_single_task_process_files.combine_files manual__2024-08-31T02:15:40.781006+00:00 [queued]>
[2024-08-31T02:16:40.756+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_single_task_process_files.combine_files manual__2024-08-31T02:15:40.781006+00:00 [queued]>
[2024-08-31T02:16:40.757+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-31T02:16:40.768+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): combine_files> on 2024-08-31 02:15:40.781006+00:00
[2024-08-31T02:16:40.774+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=629) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-31T02:16:40.776+0000] {standard_task_runner.py:63} INFO - Started process 631 to run task
[2024-08-31T02:16:40.776+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_single_task_process_files', 'combine_files', 'manual__2024-08-31T02:15:40.781006+00:00', '--job-id', '448', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp2h8ev9iq']
[2024-08-31T02:16:40.778+0000] {standard_task_runner.py:91} INFO - Job 448: Subtask combine_files
[2024-08-31T02:16:40.813+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_single_task_process_files.combine_files manual__2024-08-31T02:15:40.781006+00:00 [running]> on host 0fffd72ba499
[2024-08-31T02:16:40.884+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_single_task_process_files' AIRFLOW_CTX_TASK_ID='combine_files' AIRFLOW_CTX_EXECUTION_DATE='2024-08-31T02:15:40.781006+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-31T02:15:40.781006+00:00'
[2024-08-31T02:16:40.885+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-31T02:16:40.903+0000] {base.py:84} INFO - Using connection ID 'zillow-aws' for task execution.
[2024-08-31T02:16:40.904+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='zillow-aws', conn_type='aws') credentials retrieved from login and password.
[2024-08-31T02:18:10.569+0000] {logging_mixin.py:188} INFO - Combined file saved to zillow-staging/combined-data/combined_file.csv
[2024-08-31T02:18:10.655+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-31T02:18:10.656+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-31T02:18:10.664+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_single_task_process_files, task_id=combine_files, run_id=manual__2024-08-31T02:15:40.781006+00:00, execution_date=20240831T021540, start_date=20240831T021640, end_date=20240831T021810
[2024-08-31T02:18:10.716+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-31T02:18:10.728+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-31T02:18:10.731+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
